Disable Fastmem by default. This is an option for fast, but unsafe
memory use by the JIT that triggers SIGSEGV by design that is meant to
be caught and then replaced by the emulator. However, on OpenBSD this
just results in segfault on start of dolphin. Therefore set it to
'false' by default.

Index: Source/Core/Core/ConfigManager.cpp
--- Source/Core/Core/ConfigManager.cpp.orig
+++ Source/Core/Core/ConfigManager.cpp
@@ -479,7 +479,7 @@ void SConfig::LoadCoreSettings(IniFile& ini)
   core->Get("CPUCore", &cpu_core, PowerPC::CPUCore::Interpreter);
 #endif
   core->Get("JITFollowBranch", &bJITFollowBranch, true);
-  core->Get("Fastmem", &bFastmem, true);
+  core->Get("Fastmem", &bFastmem, false);
   core->Get("DSPHLE", &bDSPHLE, true);
   core->Get("TimingVariance", &iTimingVariance, 40);
   core->Get("CPUThread", &bCPUThread, true);
@@ -768,7 +768,7 @@ void SConfig::LoadDefaults()
   bSyncGPUOnSkipIdleHack = true;
   bRunCompareServer = false;
   bDSPHLE = true;
-  bFastmem = true;
+  bFastmem = false;
   bFPRF = false;
   bAccurateNaNs = false;
 #ifdef _M_X86_64
